### ISSUE TITLE
[Programmatic Access - Make code - Settings]: Links present on the 'About monster racer' page do not have discernible text.

### ISSUE THREAD

**Issue by vroopea** [microsoft/pxt-microbit#5495]
### User Experience:
Visually impaired people who depend on screen readers for navigation will get impacted if the screen reader does not announce the accessible name for the links. Due to which the end user will not be able to able to use the blocks.

**Note:**
User credentials should NOT be included in the bug.

### Repro Steps:

1. Launch application URL: https://arcade.makecode.com/skillmap/educator-info/racer-map-info/.
2. About monster racer page opens. Turn on the AI (Accessibility Insights).
3. Verify whether discernible text is defined for the links present on the page or not.

### Actual Result:
**Issue:** Links present on the 'About monster racer' page do not have discernible text.

**Observation:**
When focus lands on the links, the screen reader announces as "Link".

### Expected Result:
Accessible name should be defined for all the links which are present on the 'About monster racer' page.

**Example:** Aria-label="Place on random tile" should be defined.

### Issue: 
Ensures links have discernible text (link-name - [https://accessibilityinsights.io/info-examples/web/link-name](https://accessibilityinsights.io/info-examples/web/link-name))

### Target application: 
About Monster Racer - [https://arcade.makecode.com/skillmap/educator-info/racer-map-info](https://arcade.makecode.com/skillmap/educator-info/racer-map-info)

### Element path: 
table:nth-child(17) > tbody > tr:nth-child(2) > td:nth-child(2) > .link.ui:nth-child(5)

### Snippet: 
`<a class="ui link" href="/reference/tiles/place-on-random-tile">`

### How to fix: 
**Fix all of the following:**
- Element is in tab order and does not have accessible text

### Fix any of the following:

- Element does not have text that is visible to screen readers
- aria-label attribute does not exist or is empty
- aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
- Element has no title attribute

### MAS Reference:
[MAS 4.1.2 – Name, Role, Value](https://aka.ms/MAS4.1.2)

### ARIA Authoring Ref Link/Fluent Patterns Guidance /HITS Guidance:
[link-name](https://accessibilityinsights.io/info-examples/web/link-name)

### UIA/DOM details available:
Yes (Refer the attachment)

### Reference Links:

- [Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.
- External Bug Process: If this bug belongs to an external team, mark it as resolved/done and assign it back to the tester with notes on where to file/route the bug. For more information, please use this [External Bug Process (sharepoint.com)](https://microsoft.sharepoint.com/teams/cea11y/SitePages/External-Bug-Process.aspx?xsdata=MDN8MDF8fDE1MThmM2JlZGFjMDRlZDhiZmViYTUxMDA2M2NiZWI3fDcyZjk4OGJmODZmMTQxYWY5MWFiMmQ3Y2QwMTFkYjQ3fDF8MHw2Mzc3NDEyNTM2MTEzNzIwNzR8R29vZHxWR1ZoYlhOVFpXTjFjbWwwZVZObGNuWnBZMlY4ZXlKV0lqb2lNQzR3TGpBd01EQWlMQ0pRSWpvaUlpd2lRVTRpT2lJaUxDSlhWQ0k2TVRKOQ%3D%3D&sdata=MnlpSXlVMnpGT3cvZ0xJNnFWUWZWTGt6dmo1VkJMZFpIZHBvUUduNDhyMD0%3D&ovuser=72f988bf-86f1-41af-91ab-2d7cd011db47%2cv-kgeetala%40microsoft.com). “HCL Staff should not log any third-party external bugs. Should be routed to EDAD team”.
- Please reach out to [C&AI Teams channel](https://apc01.safelinks.protection.outlook.com/?url=https://teams.microsoft.com/l/team/19%253a3cd071821b694f11b054d878453cd75f%2540thread.skype/conversations?groupId%3D30e3f968-15dd-4da2-bd8e-85f27e4af2c3%26tenantId%3D72f988bf-86f1-41af-91ab-2d7cd011db47&data=02%7c01%7cAklavyaM%40hcl.com%7c24e89c6ac26f42cfe40c08d7667e49f3%7c189de737c93a4f5a8b686f4ca9941912%7c0%7c0%7c637090564906844600&sdata=22JjBMoA1ID/KnYqFa/OB%2BrkTaLh5YYsGBd/7Bm7u5w%3D&reserved=0) for any process-related queries.

### Test Environment:

- OS Version: Windows 11 23H2 (OS Build:22631.3007)
- Browser: Chrome Version 121.0.2277.4 (Official build) dev (64-bit).
- Test Matrix: Chrome + JAWS
- Tool: Accessibility Insights
- URL: https://makecode.microbit.org/.
<img width="1342" alt="MAS4 1 2_Links present on the 'About monster racer' page do not have discernible text" src="https://github.com/microsoft/pxt-microbit/assets/158465724/f2ff2e2a-1f74-44cc-ae91-80785cbc7af7">

**Comment by vroopea** [comment]
#Accessibility;#A11yMAS;#A11ySev2;#MAS4.1.2;#A11yAuto:#K4W;#HCL;#Benchmark;#Win11-Chrome;#E2E-HCL-MakeCode-Feb2024;#WCP
